tox.ini

```diff
@@ -50,6 +85,16 @@ commands =
     figure: python -c "import sunkit_image.tests.helpers as h; print(h.get_hash_library_name())"
     figure: {env:PYTEST_COMMAND} -m "mpl_image_compare" --mpl --remote-data=any --mpl-generate-summary=html --mpl-baseline-path=https://raw.githubusercontent.com/sunpy/sunpy-figure-tests/sunkit-image-main/figures/{envname}/ {posargs}
```

Comment:
This needs updating in the same way the mpl-animators one did, with the factor specific cli arguments.